shader_invert.hlsl

```diff
@@ -10,9 +10,9 @@ float4 main(float2 uv : TEXCOORD) : COLOR
         return color;
     }
 
-    color.rgb /= alpha;
-    color.rgb = 1 - color.rgb;
-    color.rgb *= alpha;
+    float3 notAlpha = color.rgb * (1.0 / max(0.0001, alpha));
+    notAlpha = 1 - notAlpha;
+    color.rgb = notAlpha * alpha;
```

Comment:
Without using `notAlpha` it gives exactly the same result, since the generated shader only preserves the logic, so I didn't bother to change it to how it was before but being more explicit, this was before for PS 2.0:
```
    color.rgb = color.rgb * (1.0 / max(0.0001, alpha));
    color.rgb = 1 - color.rgb;
    color.rgb *= alpha;
```